JourneysSquad/JourneysSquad/Model/BillValidationError.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+/// Indicates error that can be caused by incorrect input data for the bill.
+enum BillValidationError: LocalizedError {
+    /// Indicates too long participant's name.
+    case tooLongName
```

Comment:
nit:
```suggestion
    case tooLongParticipantName
```

Otherwise it might be confusing: was that too long name of the bill?

Reply:
Fixed